packages/contracts/scripts/write-constants.ts

```diff
@@ -57,6 +60,7 @@ const writeConstants = async () => {
     `pragma solidity >=0.6.2 <0.9.0;\n\n` +
     `import { NetworkInfo, NetworkType } from "./SphinxPluginTypes.sol";\n\n` +
     `contract SphinxConstants {\n` +
+    `  string public constant sphinxVersion = 'v${version}';\n` +
```

Comment:
Why is it necessary to store the contracts version in Solidity (in `sphinxVersion` here) and in TypeScript (in `contracts/src/version.ts`)? In other words, can't we just check the contracts version by doing this in the plugins package?

```ts
import { CONTRACTS_LIBRARY_VERSION } from '@sphinx-labs/contracts'

if (expectedContractsLibraryVersion !== CONTRACTS_LIRARY_VERSION) throw new Error(...)
```